### ISSUE TITLE
The format of categories does not work in most themes

### ISSUE THREAD

**Issue by RPOD** [json-editor/json-editor#165]
### Expected behavior
Properly display the categories once clicked

### Actual behavior
Only the "Basics" tab is being displayed

### Steps to reproduce the behavior
This is how I initialize / populate the editor:
```javascript
$.getJSON('data/xr_schema.json', function (json) {
    var element = document.getElementById('editor');
    editor = new JSONEditor(element, {
        theme: 'bootstrap4',
        iconlib: 'materialicons',
        schema: json,
        ajax: true
    });
    if (element) {
        $.getJSON('data/sources.json', function (source) {
            editor.setValue(source);
        });
    }
    editor.on('change', function () {
        var errors = editor.validate();
        console.log(errors);
        if (errors.length) {
            document.getElementById('badge').className = 'badge badge-danger';
            $('#badge').html('Error');
            document.getElementById('store').disabled = true;
        } else {
            document.getElementById('badge').className = 'badge badge-success';
            $('#badge').html('Valid');
            document.getElementById('store').disabled = false;
        }
    });
});
```

And this is my full schema:

```javascript
{
  "title": "Projects",
  "type": "array",
  "format": "tabs",
  "items": {
    "title": "Project",
    "type": "object",
    "format": "categories",
    "properties": {
      "name": {
        "type": "string",
        "title": "Name",
        "description": "Name of the Project",
        "minLength": 1,
        "default": ""
      },
      "ip": {
        "type": "string",
        "default": "0.0.0.0",
        "description": "The IP adress where the project is located",
        "title": "IP"
      },
      "ftp": {
        "type": "object",
        "title": "Filetransfer",
        "format": "grid",
        "properties": {
          "host": {
            "type": "string",
            "description": "The host of the FTP server"
          },
          "username": {
            "type": "string",
            "description": "The username used to log in on the FTP server"
          },
          "password": {
            "type": "string",
            "description": "The password for the ftp server to log in.",
            "format": "password"
          },
          "ignore_dirs": {
            "type": "array",
            "format": "table",
            "title": "Dirs to ignore",
            "uniqueItems": true,
            "items": {
              "type": "string",
              "description": "The full filepath on the FTP server to be ignored",
              "title": "Dir"
            }
          }
        }
      },
      "databases": {
        "type": "array",
        "title": "List of Databases",
        "uniqueItems": true,
        "format": "table",
        "items": {
          "title": "Database",
          "type": "object",
          "properties": {
            "database": {
              "type": "string",
              "description": "The databasename of the database"
            },
            "name": {
              "type": "string",
              "description": "Phonetic name of db (only for backup)"
            },
            "password": {
              "type": "string",
              "description": "The password",
              "format": "password"
            },
            "type": {
              "type": "string",
              "enum": [
                "MySQL",
                "PSQL"
              ]
            },
            "user": {
              "type": "string",
              "description": "The username for the DB"
            }
          }
        }
      }
    }
  }
}


```

**Comment by pmk65** [comment]
Doesn't seem to work in most themes. Here's a list:

| Theme       | Tab status | Comments                                                                     |
|-------------|:----------:|------------------------------------------------------------------------------|
| Barebones   |      ✓     | Works, but styling looks ugly                                                             |
| HTML        |      ✘     | Only the 1st Tab has content. For the remaining Tabs, the content is hidden. |
| jQueryUI    |      ✘     | Only the 1st Tab has content. For the remaining Tabs, the content is hidden. |
| Bootstrap2  |      ✓     | Works                                                                        |
| Bootstrap3  |      ✓     | Works                                                                        |
| Bootstrap4  |      ✘     | Only the 1st Tab has content. For the remaining Tabs, the content is hidden. |
| Foundation3 |      ✘     | Only the 1st Tab has content. For the remaining Tabs, the content is hidden. |
| Foundation4 |      ✘     | Only the 1st Tab has content. For the remaining Tabs, the content is hidden. |
| Foundation5 |      ✘     | Only the 1st Tab has content. For the remaining Tabs, the content is hidden. |
| Foundation6 |      ✓     | Works, but styling looks ugly                                                                   |
| Materialize |      ✘     | Content for each Tab is just displayed below each other.                     |

**Comment by pmk65** [comment]
See also #121

**Comment by pmk65** [comment]
The Bootstrap4 problems have been fixed in #216

**Comment by pmk65** [comment]
See #214

**Comment by pmk65** [comment]
Fixed in #243
